### PR TITLE
Add shapely as a requirement

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xmnlab** [comment]
@conda-forge-admin, please rerender

**Comment by xmnlab** [comment]
```
ibis/expr/tests/test_table.py::test_join_invalid_expr_type FAILED        [  9%]
ibis/expr/tests/test_timestamp.py::test_greater_comparison_pandas_timestamp FAILED [ 35%]
ibis/pandas/execution/tests/test_temporal.py::test_cast_datetime_strings_to_timestamp[datetime_strings_ny] FAILED [ 59%]
```

**Comment by xmnlab** [comment]
@cpcloud I just checkout the tag `1.2.0`, created the environment, ran the tests (the same used specified on meta.yaml) and the result is:

```
ibis/expr/tests/test_table.py::test_join_invalid_expr_type FAILED        [  9%]
ibis/expr/tests/test_timestamp.py::test_greater_comparison_pandas_timestamp FAILED [ 35%]
ibis/pandas/execution/tests/test_temporal.py::test_cast_datetime_strings_to_timestamp[datetime_strings_ny] FAILED [ 59%]
ibis/sql/sqlite/tests/test_functions.py::test_category_label FAILED      [ 65%]
= 4 failed, 4424 passed, 661 skipped, 3322 deselected, 232 xfailed, 49 warnings in 199.04 seconds =
```

As you can see on CI logs, there are 3 failures and pinning pandas to < 0.25, locally, the failures decrease from 3 to 1 ... My guess is that this errors could be related to some dependence incompatibility. 

not sure if I am missing something here, do you have any thoughts? 

my first guess is that maybe would be good to release a new version from master ... what do you think?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,6 +35,7 @@ requirements:
     - pydata-google-auth
     - pymapd >=0.12.0
     - pymysql
+    - pyspark >=2.4.3
```
Comment (cpcloud) [inline recipe/meta.yaml]: Do tests fail without this? I don't think we should add this as a requirement, it's pretty heavyweight and requires a JVM installation on the host machine to work properly.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -54,10 +58,12 @@ test:
     - conftest.py
     - setup.cfg
   commands:
-    - pytest --rootdir ibis -m "not (backend or bigquery or clickhouse or hdfs or impala or kudu or mapd or mysql or postgresql or superuser or udf or postgis)" --ignore ibis/tests/test_version.py --tb=line
+    - pytest --rootdir ibis -m "not (backend or bigquery or clickhouse or hdfs or impala or kudu or mapd or mysql or postgresql or superuser or udf or postgis or spark)" --ignore ibis/tests/test_version.py --tb=line
```
Comment (cpcloud) [inline recipe/meta.yaml]: You shouldn't need to do this with the commit in the recipe, since the spark backend doesn't exist at that commit.

**Review by cpcloud — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (xmnlab) [reply recipe/meta.yaml]: I just add this to see if it will change the result .. nothing special I will remove that!

**Pull request closed without merging**